### ISSUE TITLE
texturing node doesn't work with supplied UV coordinates[bug]

### ISSUE THREAD

**Issue by illusionmanager** [alicevision/meshroom#1674]
**Describe the bug**
I have created a simple object in 3dsmax, created an unwrapped UV map for it and exported it as obj file.
When I load this directly into a texturing node, the output is an object with a totally black texture.
When I edit the obj file and remove the vt lines (the texture coordinates) but leave it otherwise untouched and put that directly into a texturing node, I do get a nice cube, with a corresponding texture.
It seems that the texturing node does discover the UV's in the obj and therefore doesn't do its own unwrapping (as expected), but somehow it doesn't use them correctly and produces a black texture.
The resulting mesh DOES have the correct texture coordinates.
The real problem is that when I supply my own UV coordinates, the log file shows that NO camera is contributing to the texturing, which of course results in a black texture. As the object isn't changed I would expect the same cameras to contribute in the same way and produce a nice texture using my UV. A camera contributing to a triangle shouldn't depend on the texture coordinates but only on the coordinates, isn't it??

**Desktop (please complete the following and other pertinent information):**
 - OS: win 10

 - Meshroom version: please specify if you are using a release version or your own build

   - Binary version (if applicable) 2021.1.0

**Comment by winnechan** [comment]
hi, do you know what the **basic** algorithm meshroom use for the mesh parameterizaiton ?

**Comment by illusionmanager** [comment]
no I don't, but it is described on the web page that the documentation tab revers to. https://alicevision.org/#photogrammetry/meshing.
Any way, my problem is about texturing the mesh, not meshing

**Comment by winnechan** [comment]
Thanks at first! Mesh parameterizaiton is to create the UV index for each vertex in the mesh (UV unwarping). I would like to know how meshroom create the texture map. The ABF and LSCM are clarified in the documentation while the **Basic** mode is unknown.

**Comment by illusionmanager** [comment]
I really don't know. I'm not an expert, I just have a problem that no camera contributes to the texture maps, when I provide UV coordinates. 
From the docs it looks like the standard method they use is from Levy but I don't know at all. Does that answer your question? https://alicevision.org/#photogrammetry/texturing

**Comment by winnechan** [comment]
Yes! Thanks a lot! The post indeed answers my question. Hope your problem can be soloved A.S.A.P.
